getting-started/host_security_policy_specification.md

```diff
@@ -70,6 +70,8 @@ spec:
   action: [Audit|Block] (Block by default)
 ```
 
+**Please note that for system calls monitoring we only support audit action no matter what the value of action is**
```

Comment:
```suggestion
> **Note** system calls monitoring only supports audit action no matter what the value of action is
```

Reply:
I see this is fixed. marking it resolved